spec/system/admin/admin_updates_tag_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe "Admin updates a tag", type: :system do
+  let(:super_admin) { create(:user, :super_admin) }
+  let(:tag) { create(:tag) }
+
+  context "when no colors have been choosen for the tag" do
+    before do
+      sign_in super_admin
+      visit "/admin/tags/#{tag.id}"
+    end
+
+    it "defaults to white text for the color picker" do
+      expect(page).to have_field("tag_text_color_hex", with: "#ffffff")
+    end
+
+    it "defaults to black and white upon update" do
+      check "Supported"
+      click_button("Update Tag")
+      expect(page).to have_xpath("/html/body/div/main/header/div[1]/h2/span")
```

Comment:
**(comment)**: the test descriptions says that `it "defaults to black and white upon update"` but I'm struggling to see how this test checks that it defaults to black and white upon update

Reply:
I was unable to test that the desired CSS was showing up, so I instead checked that the `xpath` was present. Now that I'm taking another look at this, I'm wondering if I should remove this check altogether unless you or another reviewer have an idea for how to test the default better. 😊